orca-qos/src/main/kotlin/com/netflix/spinnaker/orca/qos/bufferpolicy/ApplicationConfigurationAwareSupport.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.orca.qos.bufferpolicy
+
+import com.netflix.spinnaker.orca.front50.Front50Service
+import com.netflix.spinnaker.orca.front50.model.Application
+import java.time.Clock
+import java.time.LocalDate
+
+open class ApplicationConfigurationAwareSupport(
+  private val clock: Clock,
+  private val front50Service: Front50Service
+) {
+  fun Application.quietTimeEnabled(): Boolean = this.quietTimeConfiguration().isEnabled()
```

Comment:
These extension methods should be private or internal (depending on if you need to reference them directly for tests).